cluster-provision/k8s/1.17/node01.sh

```diff
@@ -47,3 +47,4 @@ done
 
 local_volume_manifest="/tmp/local-volume.yaml"
 kubectl --kubeconfig=/etc/kubernetes/admin.conf create -f "$local_volume_manifest"
+sudo ip -6 route add default dev eth0 metric 1
```

Comment:
Why this is needed? Is there inter-network communication going on?

Reply:
without this , no ping -6 between the nodes, which is the 3rd point that Calico ipv6 explanation require from the host. 
groomed with Roman its ok and in this file, if its ok